docs/api.md

```diff
@@ -217,9 +217,10 @@ document
 | ----- | ----- | ------- |
 | paymentPointer | String | Your payment account URL. The same value is used as the `content` in your `<meta>` tag.|
 | requestId | String | This value is identical to the session ID/monetization ID (UUID v4) generated by the user agent (see [Flow](./explainer#flow)).|
-|amount | String | The destination amount received as specified in the Interledger protocol (ILP) packet. |
-|assetCode | String | The code (typically three characters) identifying the amount's unit. A unit, for example, could be a currency (USD, XRP). |
-|assetScale | Number | The number of places past the decimal for the amount. For example, if you have USD with an asset scale of two, then the minimum divisible unit is cents. |
+| amount | String | The destination amount received as specified in the Interledger protocol (ILP) packet. |
+| assetCode | String | The code (typically three characters) identifying the amount's unit. A unit, for example, could be a currency (USD, XRP). |
+| assetScale | Number | The number of places past the decimal for the amount. For example, if you have USD with an asset scale of two, then the minimum divisible unit is cents. |
+| receipt | String | base64-encoded STREAM receipt issued by the Web Monetization receiver to the Web Monetization provider as proof of the total amount received in the stream. |
```

Comment:
can we link this to the STREAM receipts spec?

Reply:
Added link (pending https://github.com/interledger/rfcs/pull/572)
https://github.com/interledger/webmonetization.org/pull/51/commits/9e265950615c8a6d88ba107115d960727a87cf2a

Reply:
> the total amount received in the stream

Interesting to note that this could be quite different to the total amount received for a given requestId (which is currently mapped 1:1 with a given meta tag)

Reply:
True, but could the client aggregate these? That seems like a better experience for the website developer

Reply:
@adrianhopebailie 
Could you elaborate/illustrate ?

Reply:
It would be nice if the browser could collect the receipts for a single requestId and make these available to the developer somehow to make it easier to sum them and get the real total for the requestId

Reply:
Receipt amounts should only be summed by the verifier, which would never be the browser.

Maybe this `receipts` field could be a list or map of the latest receipt from each STREAM connection or stream for this requestId, which could help inform a website that for some reason isn't submitting every receipts to the verifier. (If the site is only verifying every 10 receipts, they risk completely missing receipts from different connections or streams).
But I think I prefer only including the receipt corresponding the packet amount of the particular `monetizationprogress` event, in which case website should be verifying every receipt.

Reply:
>Maybe this receipts field could be a list or map of the latest receipt from each STREAM connection or stream for this requestId

That kind of grouping could be done with [`react-web-monetization`](https://github.com/sharafian/react-web-monetization) (by adding some receipt parsing) for the site.

Reply:
There might actually be cases (such as larger, discrete payments for mobile) in which we'd want to support multiple receipts in a single `monetizationprogress` (assuming `monetizationprogress` is used as opposed to a new event).